dali/kernels/alloc.cc

```diff
@@ -91,7 +92,7 @@ void Deallocate(AllocType type, void *mem, int device) noexcept {
   VALUE_SWITCH(type, type_label,
     (AllocType::Host, AllocType::Pinned, AllocType::GPU, AllocType::Unified),
     (return Allocator<type_label>::Deallocate(mem, device)),
-    (assert(!"Invalid allocation type requested");)
+    (assert(!"Invalid Deallocation type requested");)
```

Comment:
Why? There's no such thing as "deallocation type". However, it's not _requested_, so maybe:
```suggestion
    (assert(!"Invalid allocation type specified");)
```

Reply:
Missed it, will revert to how it was before.

Reply:
I misunderstood it with the function name instead of parameter name.